prbt_gazebo/launch/sim_prbt_gazebo.launch

```diff
@@ -23,6 +23,7 @@ limitations under the License.
   <arg name="headless" default="false"/>
   <arg name="debug" default="false"/>
   <arg name="pipeline" default="ompl" />
+  <arg name="blank_world" default="false" />
```

Comment:
In analogy to [empty_world.launch](https://github.com/ros-simulation/gazebo_ros_pkgs/blob/kinetic-devel/gazebo_ros/launch/empty_world.launch) you could add an argument `<arg name="world_name" default="worlds/empty.world"/>`